drush/cm_config_tools.drush.inc

```diff
@@ -108,6 +108,22 @@ function cm_config_tools_drush_command() {
     'aliases' => array('cmcr'),
   );
 
+  $items['cm-config-tools-suggest'] = array(
+    'description' => "List the dependencies of an extension's default configuration.",
+    'arguments' => array(
+      'module' => 'Module machine name.',
+    ),
+    'options' => array(
```

Comment:
Meanwhile, there's a todo note & comment above (line 95) which can be removed now :-)